#!/usr/bin/env python
PACKAGE="gen_costmap2"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_range", double_t, 0, "max obstacle range.(m)", 10., 5., 30.)
gen.add("angle_resolution", double_t, 0, "angle resolution in fake scan.(rad)", 0.02, 0.01, 0.5)
gen.add("decay", int_t, 0, "cost decay", 10, 1, 100)

exit(gen.generate(PACKAGE, "gen_costmap2", "GenCostmap2"))